rusk-wallet/src/lib/mod.rs

```diff
@@ -11,27 +11,12 @@ pub mod prompt;
 pub mod store;
 pub mod wallet;
 
-// Types that define Dusk's denomination.
-pub type Dusk = f64;
-pub type MicroDusk = u64;
+use rusk_abi::dusk::*;
 
 pub const SEED_SIZE: usize = 64;
 
-pub const ONE_MILLION: Dusk = 1_000_000.0;
+pub(crate) const MAX_CONVERTIBLE: f64 = f64::MAX / dusk(1.0) as f64;
+pub(crate) const MIN_CONVERTIBLE: f64 = from_dusk(LUX);
```

Comment:
This should be consistent. Either we have both as:
```rust
pub(crate) const MAX_CONVERTIBLE: f64 = f64::MAX / dusk(1.0) as f64;
pub(crate) const MIN_CONVERTIBLE: f64 =  LUX as f64 / dusk(1.0) as f64;
```

or:

```rust
pub(crate) const MAX_CONVERTIBLE: f64 = from_dusk(f64::MAX);
pub(crate) const MIN_CONVERTIBLE: f64 =  from_dusk(LUX);
```

If I'm not mistaken they should be equivalent.